dist/index.js

```diff
@@ -10536,7 +10534,7 @@ const createGithubAccessToken = async (githubAppToken) => {
 
   const response = await makeRequest(query, variables);
 
-  if (!response.data && !response.data.createGithubAccessToken.success) {
+  if (!response.data || !response.data.createGithubAccessToken.success) {
```

Comment:
When I execute `yarn build`, it will contain things that are not my changes.

Reply:
let me try to rebuild the master, maybe it was built not with newest version

Reply:
everything should be fine now, you should merge with newest master